src/full/Agda/Interaction/Imports.hs

```diff
@@ -1119,7 +1119,7 @@ buildInterface source fileType topLevel pragmas = do
     reportSLn "import.iface" 5 "Building interface..."
     let m = topLevelModuleName topLevel
     scope'  <- getScope
-    let scope = set scopeCurrent m scope'
+    let scope = set scopeCurrent m scope' -- TODO:: Defined but not used
```

Comment:
Can be deleted.
Maybe also `scope' <- getScope` ?

Reply:
yep.